spec/controllers/api/v1/albums_controller_spec.rb

```diff
@@ -2,21 +2,30 @@
 
 RSpec.describe Api::V1::AlbumsController, type: :controller do
   describe "GET#show" do
-    let!(:album1) {Album.create(
+
+    let!(:user) { User.create(
+      id:1,
+      email:"email6@email.com",
+      role: "member",
+      password:"123456"
+      ) }
+    let!(:album1) { Album.create(
       album: "test album 1",
       art: "https://vignette.wikia.nocookie.net/selenagomez/images/4/46/Rare_Album_Cover.jpg/revision/latest?cb=20191212172712",
       artist: "Selena Gomez",
       year: 2020,
-      genre: "Pop"
-      )}
+      genre: "Pop",
+      user: User.first
```

Comment:
Since an album should belong to a specific user, you should be testing the association to a specific user rather than just the first user returned